.editorconfig

```diff
@@ -1,4 +1,4 @@
-[*.{fs,fsx}]
+[*.{fs,fsx,ts}]
```

Comment:
this is wrong because most settings that are here don't apply to TS files but to FS

Reply:
oh, but those are prefixed with fsharp_, mmm, nevermind then, this shoudl work